cli/src/cli_util.rs

```diff
@@ -1312,6 +1325,51 @@ Then run `jj squash` to move the resolution into the conflicted commit."#,
 
         Ok(())
     }
+
+    /// Identifies branches which are eligible to be moved automatically during
+    /// `jj commit` and `jj new`. Whether a branch is eligible is determined by
+    /// its target and the user and repo config for "advance-branches".
+    ///
+    /// Returns a Vec of branches in `repo` that point to any of the `from`
+    /// commits and that are eligible to advance. The `from` commits are
+    /// typically the parents of the target commit of `jj commit` or `jj new`.
+    ///
+    /// Branches are not moved until
+    /// `WorkspaceCommandTransaction::advance_branches()` is called with the
+    /// `AdvanceableBranch`s returned by this function.
+    ///
+    /// Returns an empty `std::Vec` if no branches are eligible to advance.
+    pub fn get_advanceable_branches<'a>(
+        &self,
+        repo: &impl Repo,
```

Comment:
`repo` is known to the `WorkspaceCommandHelper`. Passing it as argument will obfuscate the scope of the `WorkspaceCommandHelper` methods.

Reply:
Thanks... not sure why I did this. I think it was a  carryover from an earlier draft and I didn't realize it wasn't necessary anymore.